docs/index.rst

```diff
@@ -3,12 +3,12 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to PlaidML's documentation!
+PlaidML
 ===================================
 
 .. toctree::
    :maxdepth: 2
-   :caption: Contents:
```

Comment:
did you mean to delete this?

Reply:
yes. I felt it was not needed.

Reply:
earlier the title was "PlaidML's Documentation" followed by the caption "contents" including "installation instructions" etc . Now the title is "PlaidML" followed by clickable links to "installation instructions" etc. I felt this was appropriate.